librz/include/rz_util/rz_table.h

```diff
@@ -55,6 +55,18 @@ typedef struct {
 
 typedef void (*RzTableSelector)(RzTableRow *acc, RzTableRow *new_row, int nth);
 
+struct rz_cons_printable_palette_t;
+
+/**
+ * \brief Visual rendering options for \ref rz_table_visual_list
+ */
+typedef struct rz_table_visual_options_t {
+	bool unicode; ///< Use Unicode runes for lines/blocks when true
+	bool color; ///< Enable ANSI colors for bars and permission strings
```

Comment:
I think `color` is not a bool. `scr.color` can take several values from 0 (black-white), 1 (16 colors), 2 (156 colors) and 3 (16M colors).